ui/apps/platform/src/Containers/NetworkGraph/common/FlowsBulkActions.tsx

```diff
@@ -7,20 +7,26 @@ type FlowsBulkActionsProps = {
     type: 'baseline' | 'active' | 'extraneous';
     selectedRows: string[];
     onClearSelectedRows: () => void;
+    markSelectedAsAnomalous?: () => void;
+    addSelectedToBaseline?: () => void;
 };
 
 function FlowsBulkActions({
     type,
     selectedRows,
     onClearSelectedRows,
+    markSelectedAsAnomalous,
+    addSelectedToBaseline,
 }: FlowsBulkActionsProps): ReactElement {
     // setter functions
-    const markSelectedAsAnomalous = () => {
+    const markSelectedAsAnomalousHandler = () => {
         // @TODO: Mark as anomalous
+        markSelectedAsAnomalous?.();
```

Comment:
Why do we need the optional chaining operator here, and immediately below?

Reply:
I added optional chaining for now because only the baselines tab is using it. The flows tab will use in a future PR. I'm using the same FlowsTable component in both sections so I needed it to be optional for now.